pkg/apis/core/types.go

```diff
@@ -3945,7 +3945,6 @@ type ServiceSpec struct {
 	// dropping the traffic if there are no local endpoints. The default value,
 	// "Cluster", uses the standard behavior of routing to all endpoints evenly
 	// (possibly modified by topology and other features).
-	// +featureGate=ServiceInternalTrafficPolicy
```

Comment:
You might need to run `make update` to update any generated files that reference this doc string